includes/uninstall.php

```diff
@@ -161,6 +161,34 @@ function delete_posts() {
 		wp_delete_post( (int) $post_id, true );
 	}
 }
+
+/**
+ * Deletes all media source terms.
+ *
+ * @since 1.10.0
+ *
+ * @return void
+ */
+function delete_terms() {
+	$taxonomy = Media::STORY_MEDIA_TAXONOMY;
+
+	$term_ids = get_terms(
+		[
+			'taxonomy'   => Media::STORY_MEDIA_TAXONOMY,
+			'hide_empty' => false,
+			'fields'     => 'ids',
+		]
+	);
+	if ( empty( $term_ids ) || ! is_array( $term_ids ) ) {
```

Comment:
This should have the same effect:

```suggestion
	if ( ! is_array( $term_ids ) ) {
```

Reply:
What if the array is empty?

Reply:
Then the foreach loop below will not do anything

Reply:
If that is the case, then why not return early then?

Reply:
It feels cleaner and is less code. But I'm indifferent.